db/migrate/20200826072722_add_profile_field_group_to_profile_field.rb

```diff
@@ -0,0 +1,7 @@
+class AddProfileFieldGroupToProfileField < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :profile_fields, :profile_field_group, index: { algorithm: :concurrently }
```

Comment:
BTW: this does not add the foreign key, only the column, you shouldp probably add a FK as well to the `profile_field_group_id`

Reply:
Excellent point, will do. Thanks!

Reply:
@rhymes I added the foreign key now. The index is probably useless as we only have 10s of profile fields with a handful of groups. We may potentially remove it in the future.